v2/README.md

```diff
@@ -11,6 +11,11 @@ Instead of deploying and managing your Azure resources separately from your Kube
 ## Project Status
 This project is stable. We follow the [Kubernetes definition of stable](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages).
 
+| Release                                                                       | Date       |                 |
```

Comment:
Can you also update the docs/hugo/content/contributing/create-a-new-release.md documentation to mention we should add the new release here?

Reply:
Done.

Reply:
Removed.